bundles/org.openhab.binding.dbquery/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="dbquery"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="query">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="influxdb2"/>
+			<bridge-type-ref id="jdbc"/>
+		</supported-bridge-type-refs>
+		<label>Query Thing</label>
+		<description>Thing that represents a native query</description>
+
+		<channels>
+			<channel id="execute" typeId="execute-channel"/>
+			<channel id="resultString" typeId="result-channel-string"/>
+			<channel id="resultNumber" typeId="result-channel-number"/>
+			<channel id="resultDateTime" typeId="result-channel-datetime"/>
+			<channel id="resultContact" typeId="result-channel-contact"/>
+			<channel id="resultSwitch" typeId="result-channel-switch"/>
+
+			<channel id="parameters" typeId="parameters-channel"/>
+			<channel id="correct" typeId="correct-channel"/>
+			<channel id="calculateParameters" typeId="calculate-parameters-channel"/>
+		</channels>
+
+
+
+		<config-description>
+			<parameter name="query" type="text" required="true">
+				<label>Query Definition</label>
+				<description>Query definition using native query language</description>
+				<context>script</context>
```

Comment:
Are you sure this context exists? It is not documented. See https://www.openhab.org/docs/developer/bindings/config-xml.html#xml-structure-for-configuration-descriptions

Reply:
Yes and it works:
https://community.openhab.org/t/is-it-possible-to-define-a-thing-parameter-to-be-multiline/102832/2

Reply:
Cool, would be awesome if you could file a PR extending the documentation!